docs/fabrikate/README.md

```diff
@@ -0,0 +1,98 @@
+```yaml
+name: azure-vote
+type: static
+method: local
+path: ./directory-you-created-with-the-azure-vote.yaml
+```
+
+Then run `$ fab install`.  You should get output like:
+```
+$ fab install
+INFO[24-07-2019 12:04:33] fab version 0.15.0
+INFO[24-07-2019 12:04:33] 🔍  Using git: /usr/bin/git
+INFO[24-07-2019 12:04:33] 🔍  Using helm: /usr/local/bin/helm
+INFO[24-07-2019 12:04:33] 🔍  Using sh: /bin/sh
+INFO[24-07-2019 12:04:33] 🔍  Using curl: /usr/bin/curl
+INFO[24-07-2019 12:04:33] 👉  Initializing Helm
+INFO[24-07-2019 12:04:33] 💾  Loading component.yaml
+INFO[24-07-2019 12:04:33] 👉  Starting install for component: azure-vote
+INFO[24-07-2019 12:04:33] 👈  Finished install for component: azure-vote
+INFO[24-07-2019 12:04:33] ✅  Installed successfully: azure-vote
+INFO[24-07-2019 12:04:33] 🙌  Finished install
+```
+
+Then run `$ fab generate`, and you should get:
+```
+$ fab generate
+INFO[24-07-2019 12:04:49] fab version 0.15.0
+INFO[24-07-2019 12:04:49] 💾  Loading component.yaml
+INFO[24-07-2019 12:04:49] 🚚  Generating component 'azure-vote' statically from path ./vote-deploy
+INFO[24-07-2019 12:04:49] 💾  Writing generated/common/azure-vote.yaml
+INFO[24-07-2019 12:04:49] 🙌  Finished generate
+```
+When these commands finish, there will be a new directory named `generated` with a subdirectory named `common` that contains the manifest file you can use to deploy the application to a Bedrock Kubernetes cluster.
+
+## Deployment
+Flux automation makes it easy to upgrade services or infrastructure deployed by Bedrock.  Flux monitors a repo designated in the deployed .tfvars file.  See [Set Up Flux Manifest Repository](../azure-simple/README.md#set-up-flux-manifest-repository) and [Set Up Terraform Deployment Variables](../azure-simple/README.md#setup-terraform-deployment-variables).
+
+## Deploy an update using Kubernetes manifest
+In this example Flux monitors the repo described under the heading [Set Up Flux Manifest Repository](../azure-simple/README.md#set-up-flux-manifest-repository).  Now add the `azure-vote` application to the running deployment by pushing the manifest proviously created by Fabrikate to the repo.  
+
+The .yaml specification describes the service `azure-vote` and type: `LoadBalancer`.  It specifies the source of the Docker image that contains it.  The container will be accessible through the load balancer.
+
+## Results
+To see the changes as Flux picks them up and deploys them, open a bash command window and navigate to the Bedrock deployment, for example, the `bedrock/cluster/environments/azure-simple` directory.
+
+Get your Flux namespace by running: `KUBECONFIG=./output/bedrock_kube_config kubectl get po --all-namespaces`
+
+Then run the command: `KUBECONFIG=./output/bedrock_kube_config kubectl logs -f flux-6ccd98596b-v4r7d --namespace=flux`.  This will display a running log of the deployment.
```

Comment:
`flux-6ccd98596b-v4r7d` is specific to your cluster and your deployment, so this will not work when somebody else tries to replicate this

to fix, you must: get the flux pod name via:

* `KUBECONFIG=./output/bedrock_kube_config kubectl get pod -n flux`
* copy the name of the pod (the one that is not memcached)

then run:

* `KUBECONFIG=./output/bedrock_kube_config kubectl logs -f flux_pod_name_from_step_above`